content/en/agent/kubernetes/control_plane.md

```diff
@@ -598,6 +599,177 @@ The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches
 
 On other managed services, such as Azure Kubernetes Service (AKS) and Google Kubernetes Engine (GKE), the user cannot access the control plane components. As a result, it is not possible to run the `kube_apiserver`, `kube_controller_manager`, `kube_scheduler`, and `etcd` checks in these environments.
 
+## Kubernetes on Rancher Kubernetes Engine (v2.5+) {#RKE}
+
+Rancher v2.5 relies on [PushProx][10] to expose control plane metric endpoints, this allows the Datadog Agent to run control plane checks and collect metrics.
+
+### Prerequisites
+
+1. Install the [rancher-monitoring chart][11]
+1. The `pushprox` daemonsets are deployed via `rancher-monitoring` and running in the `cattle-monitoring-system` namespace.
```

Comment:
Our flavor of markdown gets buggy if you leave numbered lists all the same number 😢 
```suggestion
1. Install the [rancher-monitoring chart][11].
2. The `pushprox` daemonsets are deployed with `rancher-monitoring` and running in the `cattle-monitoring-system` namespace.
```